test.js

```diff
@@ -42,6 +42,13 @@ const GROUP_ID = 'com.dummy',
         classifier: TEST_CLASSIFIER,
         generatePom: false,
         pomFile: 'existing-pom.xml'
+    },
+    TEST_CONFIG_WITH_ENV_VARIABLE = {
+        groupId: GROUP_ID,
+        repositories: [DUMMY_REPO_SNAPSHOT, DUMMY_REPO_RELEASE],
+        classifier: TEST_CLASSIFIER,
+        generatePom: false,
+        'finalName': '{name}-{{env.NODE_ENV}}.{{package.version}}',   
```

Comment:
Style: Quoting of `finalName` is unneeded, and doesn't match the rest of the area.